src/vmm/src/builder.rs

```diff
@@ -554,13 +554,27 @@ pub fn build_microvm(
         )?;
     }
 
+    // Use half of the shmem region for virtio-gpu and half for DAX
+    let _shm_region_size = arch_memory_info.shm_size >> 1;
```

Comment:
My rust is still very recent: should the underscore be removed, here?

Reply:
That silences the unused variable warning, in the case of a build with the `tee` feature enabled and the `gpu` feature disabled (which would cause it to become unused).

Reply:
Sorry for the noise!